inference-engine/tests/functional/inference_engine/serialization/single_layer/comparison.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "shared_test_classes/single_layer/comparison.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace LayerTestsDefinitions::ComparisonParams;
+
+namespace {
+    TEST_P(ComparisonLayerTest, Serialize) {
+        Serialize();
+    }
+
+std::map<std::vector<size_t>, std::vector<std::vector<size_t>>> inputShapes = {
```

Comment:
I propose to have a separate file for each operation. We can use parametrized test, but initialization of parametrized test should be in the separate file.

Reply:
Should I add this change in this PR, or create a new PR solely for it

Reply:
As this PR is already pretty messy, I would suggest to create a separate PR for this task, but it is up to you

Reply:
I propose to do it in this PR in order to don't return to revising.